jobs/resource_job_test.go

```diff
@@ -621,6 +621,130 @@ func TestResourceJobCreate_ForEachTask(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Equal(t, "789", d.Id())
 }
+func TestResourceJobCreate_PowerBiTask(t *testing.T) {
```

Comment:
Not critical, but it's best to use Mocks from Go SDK. See this PR on how it could be done: https://github.com/databricks/terraform-provider-databricks/pull/4648